### PR TITLE
Add simple dashboard to ztunnel admin

### PR DESCRIPTION
**Pull request by hanxiaop**
For convenience. Used Envoy's style.
image

### CODE REVIEW

File: src/admin.rs [hunk 1]
```diff
@@ -129,12 +130,56 @@ impl Service {
...
+    for (index, (path, description)) in apis.iter().enumerate() {
+        api_rows.push_str(&format!(
+            "<tr class=\"{row_class}\"><td class=\"home-data\"><a href=\"{path}\">{path}</a></td><td class=\"home-data\">{description}</td></tr>\n",
+            row_class = if index % 2 == 1 { "gray" } else { "vert-space" },
+            path = path,
+            description = description
+        ));
+    }
+
+    let html = fs::read("src/assets/dashboard.html").unwrap();
```
Comment (howardjohn) [inline src/admin.rs:170]: We need to embed it, this file won't exist in the docker image (and shouldn't). There are rust macros to embed

**Review by howardjohn — COMMENTED** [review]
Aren't most of these requiring POST anyways? I am not sure how much value this adds

**Comment by hanxiaop** [comment]
> Aren't most of these requiring POST anyways? I am not sure how much value this adds

I want to add this because when I try to use commands like `istioctl dashboard envoy` or maybe a new command for ztunnel, it shows an empty page. Most of the time, I'm only interested in seeing the config dump, and I'm not sure if we need to display other APIs.

**Comment by linsun** [comment]
This could be useful for dev and debugging imho

**Pull request merged**